### PR TITLE
Proc Macros + Traits vs Concepts video links

### PR DESCRIPTION
**Pull request by crowlandsimms**

### CODE REVIEW

File: draft/2022-08-24-this-week-in-rust.md [hunk 1]
```diff
@@ -36,8 +36,10 @@ and just ask the editors to select the category.
 ### Project/Tooling Updates
 
 ### Observations/Thoughts
+- [video] [Rust Traits vs C++ Concepts](https://youtu.be/f68FtmUTl3s)
```
Comment (ericseppanen) [inline draft/2022-08-24-this-week-in-rust.md:39]:
```suggestion
* [video] [Rust Traits vs C++ Concepts](https://www.youtube.com/watch?v=f68FtmUTl3s)
```

File: draft/2022-08-24-this-week-in-rust.md [hunk 2]
```diff
@@ -36,8 +36,10 @@ and just ask the editors to select the category.
...
 
 ### Rust Walkthroughs
+- [video] [Writing Procedural Macros](https://youtu.be/RfhkCdu3iYs)
```
Comment (ericseppanen) [inline draft/2022-08-24-this-week-in-rust.md:42]:
```suggestion
* [video] [Writing Procedural Macros](https://www.youtube.com/watch?v=RfhkCdu3iYs)
```

**Review by ericseppanen — APPROVED** [review]
Thanks for the links!

Note that we prefer canonical links (youtu.be redirects to youtube.com) so I've added the suggested replacements.

**Review by mariannegoldin — APPROVED** [review]
Great submissions, thanks! Approving.

**Review by mariannegoldin — APPROVED** [review]
Thanks for the submission, approving.

**Pull request merged**